.github/workflows/build.yml

```diff
@@ -557,9 +557,9 @@ jobs:
           mkdir  ${{ env.INSTALL_PORTABLE_DIR }}/lib
           cp /lib/x86_64-linux-gnu/libbz2.so.1.0 ${{ env.INSTALL_PORTABLE_DIR }}/lib
           cp /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0 ${{ env.INSTALL_PORTABLE_DIR }}/lib
-          cp /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1 ${{ env.INSTALL_PORTABLE_DIR }}/lib
-          cp /usr/lib/x86_64-linux-gnu/libssl.so.1.1 ${{ env.INSTALL_PORTABLE_DIR }}/lib
-          cp /usr/lib/x86_64-linux-gnu/libffi.so.7 ${{ env.INSTALL_PORTABLE_DIR }}/lib
+          cp /usr/lib/x86_64-linux-gnu/libcrypto.so.* ${{ env.INSTALL_PORTABLE_DIR }}/lib
+          cp /usr/lib/x86_64-linux-gnu/libssl.so.* ${{ env.INSTALL_PORTABLE_DIR }}/lib
+          cp /usr/lib/x86_64-linux-gnu/libffi.so.* ${{ env.INSTALL_PORTABLE_DIR }}/lib
```

Comment:
```suggestion
          cp /usr/lib/x86_64-linux-gnu/libffi.so.*.* ${{ env.INSTALL_PORTABLE_DIR }}/lib
```

Reply:
Why is this needed? as the original name was `/usr/lib/x86_64-linux-gnu/libffi.so.7 ` and the new one is `/usr/lib/x86_64-linux-gnu/libffi.so.8` and doesn't have double `.` (dots)

Reply:
Pretty sure the original glob will copy both libffi.so.8 and libffi.so.8.1.4. We only need one

Reply:
Pretty strange that there are two libffi files there (not sure if the inclusion of both would have hurt)